components/layouts/content-list/content-list--years.hbs

```diff
@@ -0,0 +1,79 @@
+<div class="section--bg-prim-blue">
+  <div class="content-list section__inner">
+    <h1 class="content-list__headline">Where we're at today.</h1>
+
+    <div class="content-years__preface themed-text">
+      <p>Because we don’t manufacture any goods, our carbon footprint is largely
+        driven by office utilities, employee commuting, business travel, and
+        technology systems.</p>
+
+      <p>Taking these factors into account and using the
+        <a
+          href="https://www.epa.gov/climateleadership/scope-1-and-scope-2-inventory-guidance"
+        >EPA guidelines for calculating</a>
+        carbon generation, we’ve calculated our company’s global carbon
+        footprint (including internal staff and talent) for 2019, 2020, and 2021
+        (projected):</p>
+    </div>
+
+    <dl class="content-years">
+      <dt class="content-years__title-group">
+        <h2 class="content-years__year">2019</h2>
+        <div class="content-years__stat-description">
+          <h3 class="content-years__stat">Benchmark</h3>
+        </div>
+        <hr class="content-years__connector" />
+      </dt>
+      <dd class="content-block">
+        <div class="themed-text">We generated almost
+          <span class="content-years__text-highlighted">9,000 tons</span>
+          of carbon. A substantial portion of this number is due to our staff
+          and talent commuting to offices.</div>
+      </dd><!-- .content-block -->
+    </dl><!-- .content-years -->
+
+    <dl class="content-years">
```

Comment:
Could this be set up so we only need a single `<dl>` instead of a separate list for every single stat?